system/src/Grav/Common/Twig.php

```diff
@@ -183,6 +183,10 @@ public function processPage(Page $item, $content = null)
                 $twig_vars['content'] = $content;
                 $template = $item->template() . TEMPLATE_EXT;
                 $output = $local_twig->render($template, $twig_vars);
```

Comment:
I see only one content, you sure to use the latest grav version and cache cleared?